app/client/components/user/SingleUserManagement.tsx

```diff
@@ -0,0 +1,106 @@
+import * as React from "react";
+import { connect } from "react-redux";
+
+import { editUser, resetPin } from "../../reducers/userReducer";
+import { TransferUsage } from "../../reducers/transferUsage/types";
+import { ReduxState } from "../../reducers/rootReducer";
+import EditUserForm, { ICreateUserUpdate } from "./EditUserForm";
+
+interface DispatchProps {
+  editUser: (body: any, path: any) => void;
+  resetPin: (body: any) => void;
```

Comment:
`body: {user_id: number}` might be better?

Reply:
good point